src/html5/parser.rs

```diff
@@ -3667,7 +3667,7 @@ impl<'chars> Html5Parser<'chars> {
                     let new_name = SVG_ADJUSTMENTS_ATTRIBUTES
                         .get(name)
                         .expect("svg adjustments");
-                    new_attributes.insert(new_name.to_string(), value.clone());
+                    new_attributes.insert((*new_name).to_owned(), value.clone());
```

Comment:
This seems a bit odd, i mean defer the element and then call `to_owned` if you could just call `to_string`

Reply:
Yeah, I can see that this is kind of weird, It would be fine to revert this to `to_string`. The deref is necessary because we get a `&&str` out of the hashmap. Another option that might be better to read would be this:
```rust
let &new_name = SVG_ADJUSTMENTS_ATTRIBUTES
    .get(name)
    .expect("svg adjustments");
new_attributes.insert(new_name.to_owned(), value.clone());
```
What do you think?